ReactNativeTemplate/ios/Podfile

```diff
@@ -27,6 +27,9 @@ pod 'React', :path => '../node_modules/react-native', :subspecs => [
     'RCTLinkingIOS'
 ]
 
+# NB: get rid of next line once FMDB works with xcode 9
+pod 'FMDB', :git => 'https://github.com/forcedotcom/fmdb', :branch => '2.7.2_xcode9' 
```

Comment:
Gotcha. So we leave it inlined for our workspace but use podspec from vanilla fmdb elsewhere. That seems like the best of both worlds. IIRC the fmdb project file had stuff that was missing which is another reason we compiled inline.